docroot/themes/contrib/civictheme/includes/form.inc

```diff
@@ -23,7 +23,7 @@ function civictheme_form_views_exposed_form_alter(&$form, FormStateInterface $fo
   $civictheme_listing_view_name = _civictheme_listing_get_view_name('civictheme_listing');
   // Apply filters to all views whose name starts with `civictheme_listing` or
   // a view defined in the hook.
-  if ($form_id == 'views_exposed_form' && (strpos($civictheme_listing_view_name, 'civictheme_listing') === 0 || $view->id() === $civictheme_listing_view_name)) {
+  if ($form_id == 'views_exposed_form' && (strpos($view->id(), 'civictheme_listing') === 0 || $view->id() === $civictheme_listing_view_name)) {
```

Comment:
Fixed logic, Need to compare with the current view id.

Reply:
🤦